src/components/ClassCard/ClassCard.css

```diff
@@ -0,0 +1,54 @@
+.ClassCard__wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  width: 100%;
+}
+
+.ClassCard {
+  background-color: var(--color-text-onDark-dim);
+  color: var(--color-primary);
+  padding: 1rem;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.ClassCard h2 {
+  padding-bottom: 1rem;
+}
+
+.ClassCard:active,
+.ClassCard:hover {
+  background-color: var(--color-accent-onAction);
+  color: var(--color-text-onDark);
+  transition: all 1s ease-in-out;
+  cursor: pointer;
+}
+
+.ClassCard__input {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
```

Comment:
It doesn't look like ` grid-template-rows: 1fr 1fr;` is changing something. (At least not on firefox desktop)
(`grid-template-columns: 1fr 1fr 1fr;` causes a slight change tho)

Reply:
You are right, I just changed it: